src/Tasks/Microsoft.Packaging.Tools/pkg/Microsoft.Packaging.Tools.pkgproj

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We shouldn't need this file nor the .builds file.

All these targets and tasks should just come with our SDK just like any other targets and task.